stripe/util.py

```diff
@@ -128,9 +141,38 @@ def get_object_classes():
     return OBJECT_CLASSES
 
 
+Resp = Union[StripeResponse, Dict[str, Any], List["Resp"]]
+
+
+@overload
+def convert_to_stripe_object(
+    resp: Union[StripeResponse, Dict[str, Any]],
+    api_key: Optional[str] = None,
+    stripe_version: Optional[str] = None,
+    stripe_account: Optional[str] = None,
+    params: Optional[Dict[str, Any]] = None,
+) -> "StripeObject":
+    ...
+
+
+@overload
+def convert_to_stripe_object(
+    resp: List[Resp],
```

Comment:
How is `List[Resp]` different from Resp that already is a `List[Resp]`?

Reply:
It's more specific. This overload means "if you pass a list of something in, you are guaranteed to get a list of something out", whereas the type signature in the implementation says "if you pass a Resp in (which may or may not be a list) you will get a Resp out (which may or may not be a list)".